components/browser/menu/src/main/java/mozilla/components/browser/menu/view/StickyItemLayoutManager.kt

```diff
@@ -0,0 +1,452 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.menu.view
+
+import android.content.Context
+import android.graphics.PointF
+import android.os.Parcelable
+import android.view.View
+import android.view.ViewTreeObserver
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.parcelize.Parcelize
+
+// Inspired from
+// https://github.com/qiujayen/sticky-layoutmanager/blob/b1ddb086db5b04ff3c5357dabe1bff47a935dd37/
+// sticky-layoutmanager/src/main/java/com/jay/widget/StickyHeadersLinearLayoutManager.java
+/**
```

Comment:
Is there a specific way we need to format this to give them credit? @Amejia481 might know

Reply:
Not sure what needs to be changed.

Reply:
Will change "Inspired from" to "Credit to".

Reply:
Feel free to use  "Inspired from" if you just take some ideas and create your own implementation, in the example above we ported the implementation.